medical-portal/src/API/RSBC.DMF.MedicalPortal.API/ViewModels/ChefsBundle.cs

```diff
@@ -7,4 +8,5 @@ public class ChefsBundle
     public string caseId { get; set; }
     public PatientCase patientCase { get; set; }
     public Driver driverInfo { get; set; }
+    public IEnumerable<MedicalConditionItem> medicalConditions { get; set; }
```

Comment:
MedicalConditionItem is a proto model. Proto models should only be used when transferring DTOs from gRPC services e.g. cms-adapter. I have a ViewModel I can send you, it's just a straight copy of your previous MedicalCondition class in the old CaseService.cs file.

Reply:
`
namespace RSBC.DMF.MedicalPortal.API.ViewModels
{
    public class MedicalCondition
    {
        public string Id { get; set; }
        public string Description { get; set; }
        public string FormId { get; set; }
    }
}
`

Reply:
Done, thanks!